views/partials/vpn-banner.hbs

```diff
@@ -0,0 +1,42 @@
+<aside class="vpn-banner">
+	{{#with (vpnBannerData)}}
+	<section class="vpn-banner-top">
+		<label>{{ getString "vpn-banner-location" }} <output>{{ shortLocation }}</output>&nbsp; • </label>
```

Comment:
This is a lot of hard-coded characters: can we have the localization exposed as part of the string?

Reply:
Do you mean `{{ shortLocation }}`?  It is derived from a 3rd party (Maxxmind).  Else, if you mean `&nbsp; • `, yes – that could likely be added to the .ftl if you prefer.

Reply:
```
vpn-banner-location = Your location: <output>{ $location}</output>
```

BTW, why do you need the `&nbsp; •`? Are they all on the same line? I looked at Figma, and can't see anything similar. 

Do you have a screenshot of the result of this patch? The layout looks all but localization friendly.

Reply:
The `&nbsp; • ` is all on the same line – see top bar of Figma file after location.  I refactored this into the CSS instead; the resulting string and template should be cleaner now.